ride-share.rb

```diff
@@ -0,0 +1,144 @@
+RIDE_SHARE = {
+    'DR0004': [
+        {
+            date: '3rd Feb 2016',
```

Comment:
Looks like your editor is set up to use 4-space tabs. Please adjust it to use 2-spaces, and let an instructor know if you need a hand.

There's not a big reason to choose one over the other, but it is important that we all be using the same style, especially once we start working in groups.